charts/comet-api-v1/templates/deployment.yaml

```diff
@@ -71,6 +71,9 @@ spec:
           volumeMounts:
             - mountPath: /tmp
               name: tmp
+            - mountPath: "/var/run/secrets/kubernetes.io/serviceaccount"
+              name: service-account
+              readOnly: true
```

Comment:
I think the if for service account is missing here?
{{- if .Values.serviceAccount.create }}

Reply:
362addcdf89acc508bc021634f78136598364570